Intersect.Client.Framework/Entities/IAnimation.cs

```diff
@@ -3,24 +3,17 @@
 
 namespace Intersect.Client.Framework.Entities
 {
-    public interface IAnimation
+    public interface IAnimation : IDisposable
     {
         bool AutoRotate { get; set; }
         bool Hidden { get; set; }
         bool InfiniteLoop { get; set; }
         AnimationBase MyBase { get; set; }
+        Point Size { get; }
 
-        Point AnimationSize();
-        void Dispose();
-        bool Disposed();
-        void DisposeNextDraw();
```

Comment:
I actually would have kept this, but if you want this gone that's fine by me.

Reply:
Still there, just renamed as Size.